tube-player/modules/08-Add-API-endpoints/README.md

```diff
@@ -50,6 +50,7 @@ Add the following namespace to the *GlobalUsings.cs* file:
 ```csharp
 global using Refit;
 global using TubePlayer.Services;
+global using Uno.Extensions.Http;
```

Comment:
Not coming by default with the template with the options used for TubePlayer ?

Reply:
@kazo0 
Answer from @nickrandolph :
"http should be added when adding the packages, I think perhaps the app needs to be rebuilt"

Can you verify this please

Reply:
The HTTP option is not used by the template, it is added manually later on in the workshop

Reply:
hmm ill run through this again, but it wasn't building after following the steps in the module until I added this to the global usings

Reply:
keep us updated please